src/store/modules/images.js

```diff
@@ -430,14 +430,19 @@ const actions = {
     for (let channel = 0; channel < 3; channel++) {
       const url = base_url + `/infoimage/${site}/${channel + 1}`
       axios.get(url).then(response => {
+        // Handle case if no info image exists
```

Comment:
Do we have any plans to actually use info images yet?
I had been wondering if it was just better to remove info images if the feature will never get used.

Reply:
I'm not sure, but this update (along with recent changes in photonranch-api) modifies the http response when there's no info images from a 404 (not found) to a 204 (no content) which removes the errors in the browser console. That should help keep this feature entirely out of the way when it's not in use.